### PR TITLE
Specify `client_ttl` details in name entry

### PR DESCRIPTION
**Pull request by davidyuk**
This PR is supported by the Æternity Foundation

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (hanssv) [inline .vscode/settings.json:1]: I suggest putting this in your local git exclude?

File: serializations.md [hunk 2]
```diff
@@ -404,7 +404,7 @@ object (to which tag and version need to be prepended) are:
 , <code>       :: binary()
 , <log>        :: binary(),
 , <active>     :: bool(),
-, <referers>   :: [id()],
```
Comment (hanssv) [inline serializations.md:407]: aligning the `::` might look better

**Review by hanssv — COMMENTED** [review]
Good stuff!

File: serializations.md (hunk 2 above)
Comment (davidyuk) [reply serializations.md:407]: Sure! Missed that

File: .vscode/settings.json (hunk 1 above)
Comment (davidyuk) [reply .vscode/settings.json]: The idea was to help somebody who will do the same using vscode and [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker), but I removed that 👌

**Review by hanssv — APPROVED** [review]

**Review by ThomasArts — APPROVED** [review]

**Pull request merged**